docs/dev/stream/state/state.md

```diff
@@ -400,6 +400,37 @@ This option is not applicable for the incremental checkpointing in the RocksDB s
 - For existing jobs, this cleanup strategy can be activated or deactivated anytime in `StateTtlConfig`, 
 e.g. after restart from savepoint.
 
+#### Cleanup in background
+
+Besides cleanup in full snapshot, you also can activate the cleanup in background. The option will adapt
```

Comment:
`Besides cleanup in full snapshot, you can also activate the cleanup in background. The following option will activate a default background cleanup in `StateTtlConfig` if it is supported for the used backend:`

then code example